doc/api/process.markdown

```diff
@@ -238,8 +238,8 @@ A `Writable Stream` to `stdout` (on fd `1`).
 
 Example: the definition of `console.log`
 
-    console.log = function(d) {
-      process.stdout.write(d + '\n');
+    Console.prototype.log = function() {
+      this._stdout.write(util.format.apply(this, arguments) + '\n');
```

Comment:
I think we should leave it as is. Maybe reword caption to make it clear that it is not how actual `console.log` is implemented, but an example of how it could be implemented